entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_request.proto

```diff
@@ -59,6 +59,7 @@ message AttributeFilter {
   Operator operator = 2;
   AttributeValue attributeValue = 3;
   repeated AttributeFilter childFilter = 4;
+  bool is_multi_valued = 5;
```

Comment:
We should first publish the proto API separately, fix the consumers to send the correct boolean flag, otherwise, merging this PR would mean that labels filter would start failing, because I am guessing `is_multi_valued` will be false by default

Reply:
is_multi_valued will be fetched from AttributeService for labels. Please check EntityAttributeMapping.

Reply:
I take that back, I realize since it's using attribute service, it should be fine, unless there are consumers of EDS which appliers on labels